release-notes/ugandaemr-310.md

```diff
@@ -1,30 +1,36 @@
-# 3.1.0 - September 2020 
+# 3.1.0 - October 2020 
 
 ## New Features
 
 1. Clinical Workflows
    * Added TPT status to patient dashboard
    * Added ability to restrict return visit dates not to happen on public holidays and to limit the number of patients who have an appointment on a specific date 
-2. ART Clinical Assessment - the clinican 
-   * TX\_ML Report
-3. Pharmacy and Dispensing
+   * TB Treatment Support using both Point of Care and Restrospective
+   * Transfer in and out forms
+   * Review of DSD Models.
```

Comment:
@solemabrothers  can we give more details on what has been reviewed in DSDM models